src/app/components/VisualImage.js

```diff
@@ -0,0 +1,26 @@
+function VisualImage(imageSource) {
+  // Checking that img source is in base64 format
+  // FIXME: If we should use only base64/jpg, delete one condition in if 
+  const reg = /^data:image\/(?:gif|png|jpeg|bmp|webp)(?:;charset=utf-8)?;base64,(?:[A-Za-z0-9]|[+/])+={0,2}/;
+  if(reg.test(imageSource) || /.jpg/.test(imageSource)) {
+  // Creating div and img
+  const createDiv = document.createElement("div");
+  createDiv.className = "visual__box";
+
+  const createImg = document.createElement("img");
+  createImg.classList.add("visual__image");
```

Comment:
So according to above, if it's parent would have `visual-box` class, this one should have `visual-box__image`